File: mvn-advent-of-code/src/test/java/advent/twenty_fifteen/Day02IWasToldThereWouldBeNoMathTest.java
package advent.twenty_fifteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day02IWasToldThereWouldBeNoMathTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day02IWasToldThereWouldBeNoMath("/twenty_fifteen/Day02-IWasToldThereWouldBeNoMath-input.txt");

        assertEquals(1606483, dp.getPart1Answer().intValue());
        assertEquals(3842356, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day02IWasToldThereWouldBeNoMath("/twenty_fifteen/Day02-IWasToldThereWouldBeNoMath-example#1.txt");

        assertEquals(101, dp.getPart1Answer().intValue());
        assertEquals(48, dp.getPart2Answer().intValue());
    }
}

File: go-advent-of-code/twenty_fifteen/day02_I_was_told_there_would_be_no_math_test.go
package twenty_fifteen

import "testing"

func TestDay02_Real(t *testing.T) {
	part1,part2 := Day02("../data/twenty_fifteen/Day02-IWasToldThereWouldBeNoMath-input.txt")

	assertEquals(t, 1606483, part1)
	assertEquals(t, 3842356, part2 )
}

